ra-optimisation/linear-range-action-rao/src/main/java/com/farao_community/farao/linear_range_action_rao/LinearRangeActionRao.java

```diff
@@ -35,7 +51,60 @@ public String getVersion() {
 
     @Override
     public CompletableFuture<RaoComputationResult> run(Network network, Crac crac, String variantId, ComputationManager computationManager, RaoParameters parameters) {
-        // TODO : implement LinearRangeActionRao
         return null;
     }
+
+    public CompletableFuture<RaoComputationResult> run(Network network, Crac crac, String variantId,
+                                                       ComputationManager computationManager, RaoParameters parameters,
+                                                       SensitivityComputationFactory sensitivityComputationFactory) {
+        SensitivityComputationService.init(sensitivityComputationFactory, computationManager);
```

Comment:
Shouldn't this be inside the SensitivitySecurityAnalysisService ? It's weird to initialize the SensitivityComputationService when it seems that you're not using it afterward.

Reply:
Thanks. This is yet another striking example of to what point I do not understand the architecture (at all).
Changed init().